x/protorev/genesis.go

```diff
@@ -42,32 +42,29 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	}
 	k.SetPoolWeights(ctx, poolWeights)
 
-	// Configure the initial base denoms used for cyclic route building
-	baseDenomPriorities := []*types.BaseDenom{
+	// Configure the initial base denoms used for cyclic route building. The order of the list of base
+	// denoms is the order in which routes will be prioritized i.e. routes will be built and simulated in a
+	// first come first serve basis that is based on the order of the base denoms.
+	baseDenoms := []*types.BaseDenom{
 		{
 			Denom:    types.OsmosisDenomination,
 			StepSize: sdk.NewInt(1_000_000),
 		},
 	}
-	if err := k.SetBaseDenoms(ctx, baseDenomPriorities); err != nil {
+	if err := k.SetBaseDenoms(ctx, baseDenoms); err != nil {
 		panic(err)
 	}
 
-	// Update the pools on genesis
-	if err := k.UpdatePools(ctx); err != nil {
+	// Setting the admin account by default to a trusted address
```

Comment:
Can we document the source of this address? (Perhaps link to the commonwealth post once it exists)

Reply:
Yup, will do. Just for reference this is an account we created and are storing on a ledger. We will use it to enable authz (which we have already test) for a different account so that the admin keys are safely stored.